source/_integrations/homekit.markdown

```diff
@@ -358,7 +358,14 @@ homekit:
 ```
 
 Restart your Home Assistant instance. This feature requires running an mDNS forwarder on your Docker host, e.g., `avahi-daemon` in reflector mode. This kind of setup most likely requires `safe_mode` during the bridge setup.
- 
+
+## Firewall
+
+If you have a firewall configured on your Home Assistant system, make sure you open the following ports:
+
+- UDP: 5353 
+- TCP: 51827
```

Comment:
This is not correct. The TCP port is not a fixed port.

Reply:
It is a configured port, see https://www.home-assistant.io/integrations/homekit/#configuration-variables

Reply:
Yes, that is why it is not "fixed" as this example/comment does imply.
Furthermore, for 0.110, the HomeKit moves to the UI configuration flow, which allows for multiple HomeKit integration instances which makes for multiple ports even.